tests/lcs/core/math/math.livecodescript

```diff
@@ -918,3 +918,37 @@ repeat with i = 1 to 10
 end repeat
 
 end TestMathArrayWrapArray
+
+on _RandomMaxLimitTestThrows
+
+	local tN1, tN2
+
+	put 2^53 into tN1
```

Comment:
I think 2^53 should be fine as that is the maximum number we can represent - so you need to use the next representable double *after* 2^53 to trigger a throw.